src/Database/DBStructure.php

```diff
@@ -1277,11 +1277,12 @@ public static function definition() {
 		$database["oembed"] = array(
 				"fields" => array(
 						"url" => array("type" => "varbinary(255)", "not null" => "1", "primary" => "1"),
+						"maxwidth" => array("type" => "int(11)", "not null" => "1", "primary" => "1"),
```

Comment:
Please add some docu to https://github.com/friendica/friendica/blob/develop/doc/database/db_oembed.md

Reply:
Done.